.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: "StashRB: Test"
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+jobs:
+  run-specs:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # Due to https://github.com/actions/runner/issues/849, we have to use quotes for '3.0'
```

Comment:
Reminder for myself: also fix this in ActiveStash CI

Reply:
nice find 👍🏼

Reply:
Cheers!

Related PR in ActiveStash: https://github.com/cipherstash/activestash/pull/60